electron-1/main.js

```diff
@@ -15,10 +16,14 @@ function createWindow() {
       // ------
       // You are not allowed to change the value
       // of this 3 flags
+      // ok
       nodeIntegration: false,
       enableRemoteModule: false,
-      additionalArguments: []
+      additionalArguments: [],
       // ------
+
+      // found path.join here https://github.com/electron/electron/issues/4459
```

Comment:
It is actually specified in the [BrowserWindow documentation](https://electronjs.org/docs/api/browser-window)
> The value should be the absolute file path to the script